README.md

```diff
@@ -0,0 +1,140 @@
+      aggregate_translator: 'Prooph\EventSourcing\EventStoreIntegration\AggregateTranslator'
+```
+
+Here is an example of the corresponding service XML configuration with container-interop for the proophessor-do example.
+
+```xml
+<?xml version="1.0" encoding="UTF-8" ?>
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <services>
+
+        <service id="Prooph\ProophessorDo\Model\User\Handler\RegisterUserHandler.factory"
+                 class="Prooph\ProophessorDo\Container\Model\User\RegisterUserHandlerFactory"/>
+        <service id="Prooph\ProophessorDo\Model\User\Handler\RegisterUserHandler" class="Prooph\ProophessorDo\Model\User\Handler\RegisterUserHandler">
+            <factory service="Prooph\ProophessorDo\Model\User\Handler\RegisterUserHandler.factory" method="__invoke"/>
+            <argument type="service" id="interop_container"/>
+        </service>
+
+        <service id="Prooph\ProophessorDo\Model\User\UserCollection.factory"
+                 class="Prooph\ProophessorDo\Container\Infrastructure\Repository\EventStoreUserCollectionFactory"/>
+        <service id="Prooph\ProophessorDo\Model\User\UserCollection" class="Prooph\ProophessorDo\Model\User\UserCollection">
+            <factory service="Prooph\ProophessorDo\Model\User\UserCollection.factory" method="__invoke"/>
+            <argument type="service" id="interop_container"/>
+        </service>
+
+        <service id="Prooph\ProophessorDo\Projection\User\UserProjector.factory"
+                 class="Prooph\ProophessorDo\Container\Projection\User\UserProjectorFactory"/>
+        <service id="Prooph\ProophessorDo\Projection\User\UserProjector" class="Prooph\ProophessorDo\Projection\User\UserProjector">
+            <factory service="Prooph\ProophessorDo\Projection\User\UserProjector.factory" method="__invoke"/>
+            <argument type="service" id="interop_container"/>
+        </service>
+
+        <service id="Prooph\ProophessorDo\Projection\User\UserFinder.factory"
+                 class="Prooph\ProophessorDo\Container\Projection\User\UserFinderFactory"/>
+        <service id="Prooph\ProophessorDo\Projection\User\UserFinder" class="Prooph\ProophessorDo\Projection\User\UserFinder">
+            <factory service="Prooph\ProophessorDo\Projection\User\UserFinder.factory" method="__invoke"/>
+            <argument type="service" id="interop_container"/>
+        </service>
+
+    </services>
+</container>
+```
+
+Here is an example how to call the `RegisterUser` command:
+
+```php
+    /* @var $container \Symfony\Component\DependencyInjection\ContainerBuilder */
+    $commandBus = $container->get('prooph.service_bus.command_bus');
+    $messageFactory = new \Prooph\Common\Messaging\FQCNMessageFactory();
```

Comment:
We should not use the message factory here as you can instantiate the command by simply doing: `$command = new \Prooph\ProophessorDo\Model\User\Command\RegisterUser([/*payload here*/])`